src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -415,6 +415,8 @@ public static void verifyInputSchemaMatchesOutputSchema(Map<String, String> outp
       index++;
     }
 
+    logger.info("Actual input schema: " + actualInputSchema + "  Output schema: " + outputSchema);
```

Comment:
nit: 

`Output schema` -> `Expected Output Schema`

`Actual input schema` -> `Actual Input Schema`

Reply:
Have you verified locally? I think it will print the object hash of map rather than the content.